website/src/models/user.js

```diff
@@ -390,6 +390,15 @@ var UserSchema = new Schema({
       hatchPet: {type: Boolean, 'default': false},
       raisePet: {type: Boolean, 'default': false},
       streak: {type: Boolean, 'default': false}
+    },
+    improvementCategories: {
```

Comment:
I forgot this was under the preferences object, which is _mostly_ fine to update via the put route.

The problem with the user PUT route is that the user has some protected attributes that should not be allowed to be changed (gem balance, for instance), and some that can be (preferences, for instance).

The tricky thing is that there are some preferences, like background choice, which need to be protected. 

It's probably fine to leave this on the put route, but I do think eventually (v3) we're going to want a separate preferences route and move anything off the preferences object that can't be freely updated.